stream-chat-android/src/main/java/com/getstream/sdk/chat/viewmodel/messages/MessageListItemLiveData.kt

```diff
@@ -81,19 +82,19 @@ internal class MessageListItemLiveData(
     }
 
     @UiThread
-    internal fun messagesChanged(messages: List<Message>) {
-        messageItemsBase = groupMessages(messages)
+    internal fun messagesChanged(messagesUpdate: MessagesUpdate) {
+        messageItemsBase = groupMessages(messagesUpdate.messages)
         messageItemsWithReads = addReads(messageItemsBase, readsLd.value)
         val out = messageItemsWithReads + typingItems
-        val wrapped = wrapMessages(out, hasNewMessages)
+        val wrapped = wrapMessages(out, hasNewMessages, messagesUpdate.isLoadingMore)
```

Comment:
Not related to this PR, but why do we reconstruct MessageListItemWrapper every time instead of reducing its state? E.g. `value = value.copy(items = newItems)`